packages/owc-dev-server/README.md

```diff
@@ -33,26 +33,41 @@ Please note that the owc-dev-server *only resolves bare module specifiers*, and
 :::
 
 
-### Usage
+## Usage
+If you just want to try owc-dev-server once, you can [run it through `npx`](#execute-with-npx). If you decide to keep using owc-dev-server in your project, you should install and add it to your package's `devDependencies`:
 ```bash
 npm i -D owc-dev-server
+# or
+yarn add --dev owc-dev-server
 ```
 
-#### Execute
-```
+Note: The owc-dev-server package is *named after the command* it provides. That's why it doesn't follow the naming pattern for the other open-wc packages and does not use a `@open-wc/` scope prefix.
+
+### Execute as script
+For convenience (i.e. to always use the same [command line options](#command-line-options)) you should add an entry to your `scripts`:
+```javascript
 // package.json
 "scripts": {
   "start": "owc-dev-server"
 }
+```
 
-// bash
+This allows you to execute owc-dev-server like this:
+```bash
+# run package.json script 'start'
 npm run start
+# or
+yarn run start
```

Comment:
```suggestion
yarn start
```

yarn does not need the run

Reply:
I know that the `run` is optional. It's my personal preference, though.
This way, if `yarn` ever comes up with a new sub-command that happens to be the same as one of my scripts, I won't have to change my habits -- or worse: documentation.